diagnostics/ENSO_MSE/COMPOSITE/get_flux_clima.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np

```

Comment:
get_flux_in, get_flux_clima, get_flux_in24 are very similar, and can probably combined